app/views/shared/_publisher_metadata.html.erb

```diff
@@ -0,0 +1,16 @@
+<%
+  add_stylesheet_path("helpers/_publisher-metadata-with-logo.css")
+%>
+
+<div class="govuk-grid-row govuk-!-margin-top-3">
```

Comment:
Do we need the class `govuk-!-margin-top-3` here? It adds some additional spacing that the page doesn't have in government-frontend